workers/unity/Assets/Tests/EditmodeTests/Correctness/Core/OpOrderingTests.cs

```diff
@@ -0,0 +1,79 @@
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Test;
+using Improbable.Gdk.TestUtils;
+using Improbable.Worker.CInterop;
+using NUnit.Framework;
+
+namespace Improbable.Gdk.EditmodeTests
+{
+    [TestFixture]
+    public class Test : MockBase
+    {
+        private const long EntityId = 1;
```

Comment:
Why do you need this if you just used the literal `1`? 😂

Reply:
:this:

Reply:
Woops, fixed!